store/postgres/asset_repository.go

```diff
@@ -0,0 +1,401 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/odpf/columbus/asset"
+	"github.com/odpf/columbus/user"
+)
+
+const (
+	DEFAULT_MAX_SIZE = 100
```

Comment:
```suggestion
	MAX_RESULT_SIZE = 100
	MAX_DEFAULT_SIZE = 30
```

I think we should have the number above
MAX_RESULT_SIZE -> to handle if user input size > MAX_RESULT_SIZE, we limit it to MAX_RESULT_SIZE
MAX_DEFAULT_SIZE -> if user doesn't put the size, so checking it will be if size < 1

And this could go to `postgres.go` file so other repository could use the same value.

Reply:
hmm do you think we should we limit maximum result size for now?

And for the default max size, `MAX_DEFAULT_SIZE` has different meaning.
It means more to `The maximum value for default size`.

On the other hand, `DEFAULT_MAX_SIZE` means `The default value for max size`. This is what you aim for right with the constant?

Reply:
Oh yes sorry, what I meant is.

```suggestion
	DEFAULT_MAX_RESULT_SIZE = 100
	DEFAULT_RESULT_SIZE = 30
```
My concern is, there will be a plenty of assets. If our API is being used internally, it is not necessary I think to limit it. But if it will be used externally (assuming there is bad actors), not limiting the result size could impact the performance.

Hmm, let's just use `The default value for max size` for now. But instead of calling it `DEFAULT_MAX_SIZE`, can we make the name clearer by using `DEFAULT_MAX_RESULT_SIZE`.
We can